packages/gatsby/src/commands/build-html.ts

```diff
@@ -127,14 +138,18 @@ const buildPages = async ({
   pagePaths,
   activity,
   workerPool,
-}) => {
-  const rendererPath = await buildRenderer(program, stage, {
-    parentSpan: activity.span,
-  })
-  await doBuildPages({ rendererPath, pagePaths, activity, workerPool })
+}: {
+  program: IProgram
+  stage: BuildHTMLStage
+  pagePaths: string[]
+  activity: IActivity
+  workerPool: IWorkerPool
+}): Promise<void> => {
+  const rendererPath = await buildRenderer(program, stage, activity.span)
+  await doBuildPages(rendererPath, pagePaths, activity, workerPool)
   await deleteRenderer(rendererPath)
 }
 
-module.exports = {
+export default {
```

Comment:
This feels ~~a bit icky~~ non-idiomatic. If you change this to a named export, you could just do a named import in build.ts and develop.ts.